packages/client/components/TaskEditor/CommentEditor.tsx

```diff
@@ -189,6 +193,16 @@ const CommentEditor = (props: Props) => {
     onBlur(e)
   }
 
+  useEffect(() => {
+    if (!autofocus) {
+      return
+    }
+
+    if (editorRef.current) {
+      editorRef.current.focus()
+    }
+  }, [editorRef, discussionId, autofocus])
```

Comment:
+1 no need to put `editorRef` in deps array, it won't trigger a rerender anyway. technically it should trigger `exhuastive-deps` eslint warning